server/server.go

```diff
@@ -211,6 +212,11 @@ func (s *Server) githubEvent(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if pingEvent != nil {
```

Comment:
If you keep this check here the `s.handleIssueEvent(event)` will never run again.

you need to check if the header of the request contains the `X-GitHub-Event: Ping` as well

the ping event only happens once, when you create the webhook. after that this event will not be trigger

Reply:
Thank you for finding this one. 
Now the server throws a nullpointer exception in https://github.com/mattermost/mattermost-mattermod/blob/3e19389a41ff7a4c2d13e618a8d187a5083832cb/server/issue.go#L20 while trying to split the HTMLURL:
`parts := strings.Split(*event.Issue.HTMLURL, "/")`

I guess it is a local config problem. Anyway the `s.handleIssueEvent(event)` is reachable again now.

Reply:
just add this guard here: https://github.com/mattermost/mattermost-mattermod/blob/3e19389a41ff7a4c2d13e618a8d187a5083832cb/server/issue.go#L17
```
if event == nil || event.Issue == nil {
		return
	}
```
tested locally and it fix the panic
this is because when you setup the webhook you set the event pushes to be sent and this event there is no Issue data in that. but this is a good check to make it.
thanks so much for this PR :)

Reply:
Added, thank you.